src/robusta/integrations/prometheus/models.py

```diff
@@ -20,15 +20,32 @@
     StatefulSetEvent,
 )
 from robusta.integrations.kubernetes.custom_models import RobustaDeployment, RobustaJob, RobustaPod
+from robusta.model.global_config import GlobalConfig
 
 SEVERITY_MAP = {
     "critical": FindingSeverity.HIGH,
+    "high": FindingSeverity.HIGH,
+    "medium": FindingSeverity.MEDIUM,
     "error": FindingSeverity.MEDIUM,
     "warning": FindingSeverity.LOW,
+    "low": FindingSeverity.LOW,
     "info": FindingSeverity.INFO,
+    "debug": FindingSeverity.DEBUG,
 }
 
 
+class SeverityMapper:
+    @staticmethod
+    def get(value: str, default: FindingSeverity = FindingSeverity.INFO) -> FindingSeverity:
```

Comment:
Can this be simplified?

Instead of using the GlobalConfig object, can we just add the custom mapping to the SEVERITY_MAP once when we reload config?